__tests__/app.test.js

```diff
@@ -63,6 +63,7 @@ describe('GET review by id', () => {
           votes: expect.any(Number),
           created_at: expect.any(String),
           owner: expect.any(String),
+          comment_count: '3',
```

Comment:
comment_count should be a number, have a look at how to cast one data type to another in an SQL query